CONTRIBUTING_MODELS.md

```diff
@@ -22,29 +22,29 @@ Keep this checklist handy!
 
 ### Step 2: PR #1 - Add XXBackbone
 
-- [ ] An `xx/xx_backbone.py` file which has the model graph \[[Example](https://github.com/keras-team/keras-nlp/blob/master/keras_nlp/models/distil_bert/distil_bert_backbone.py)\].
-- [ ] An `xx/xx_backbone_test.py` file which has unit tests for the backbone \[[Example](https://github.com/keras-team/keras-nlp/blob/master/keras_nlp/models/distil_bert/distil_bert_backbone_test.py)\].
+- [ ] An `xx/xx_backbone.py` file which has the model graph \[[Example](https://github.com/keras-team/keras-nlp/blob/master/keras_hub/models/distil_bert/distil_bert_backbone.py)\].
```

Comment:
Unrelated to this PR: the URL is broken (probably since we added `api` and `src` dirs)!